devel/util-linux/Portfile

```diff
@@ -100,4 +102,21 @@ destroot {
                        ${destroot}${docdir}
 }
 
-github.livecheck.regex  {([^"rc]+)}
+github.livecheck.regex  {([^\"rc]+)}
+
+subport libuuid-util-linux {
+    license                 BSD
+    description             Universally unique id library
+    long_description        ${description} from util-linux.
+    conflicts               ossp-uuid libuuid
+    configure.args          --disable-all-programs \
+                            --enable-libuuid \
+                            --disable-bash-completion
+    build.target            libuuid
+    destroot {
+        system -W ${worksrcpath} "make install DESTDIR=${destroot}"
```

Comment:
Why is this needed? Isn't this what the destroot phase would do normally if you didn't override it? So why override it? Why not put the `file delete --force`s below (which could be simplified to just `delete`s) in a `post-destroot` block? Put all of the code above this subport which pertains only to the main port inside an `if {${subport} ne ${namme}} { ... }` block.